README.md

```diff
@@ -111,17 +111,21 @@ NOTE: If using a virtual beacon provider such as Mist or Beacon Point with PWMan
 
 The `PWVBLocationManager` class is the parent class to both flavors of virtual beacon location managers: `PWBPLocationManager` (Beacon Point) and `PWMSLocationManager` (Mist). They are both created with similar functions which require an virtual beacon SDK Token, a floor configuration array, and a completion block.
 
-The floor configurations array contains a configuration dictionary for each floor. The dictionaries contain a virtual beacon floor name `(NSString *)`, a matching MaaS floor identifier `(NSNumber *)`, an upper left coordinate dictionary, and a lower right coordinate dictionary. The upper left and lower right coordinate dictionaries contain latitude/longitude coordinates that match to the origin (0,0) of the virtual beacon map and the maximum (x,y) position of the virtual beacon map, respectively. Upper left matches to (0,0), and lower right matches to (max(x),max(y)). Example floor configuration dictionary:
+The floor configurations array contains a configuration dictionary for each floor. The dictionaries contain a virtual beacon floor name `(NSString *)`, a matching MaaS floor identifier `(NSNumber *)`, an upper left coordinate dictionary, and a lower right coordinate dictionary. The top left, top right, and bottom left coordinate dictionaries contain latitude/longitude coordinates that match to the top left, the top right, and the bottom left of the virtual beacon map, respectively. Example floor configuration dictionary:
```

Comment:
We are using **top** left and **bottom** left in the code and else where in the comment, guess we can unify it by not using top left and lower right